src/main/resources/studentEmailFragment-googleIdReset.html

```diff
@@ -1,18 +1,25 @@
-<p/>
-As requested, we have reset the link from your TEAMMATES account for the course <i>${courseName}</i> to its previous Google account.
+<p>
+    As requested, we have reset the link from your TEAMMATES account 
+    for the course <i>${courseName}</i> to its previous Google account.
+</p>
 
-<p/>
-<strong>If you wish to continue to access TEAMMATES using your Google account for this course, please rejoin this course through this Web address: </strong><a href="${joinUrl}">${joinUrl}</a>
-<br>*If prompted to log in, use your Google account to log in. If you do not have a Google account, please create one from the <a href="https://accounts.google.com/NewAccount">Google Accounts page</a>. 
-<br>*The above link is unique to you. Please do not share it with your classmates.
+<p>
+    <strong>If you wish to continue to access TEAMMATES using your Google account for this course, 
+        please rejoin this course through this Web address: </strong>
```

Comment:
There are currently two different ways of using the `<strong>` element in this file, one where `</strong>` is at the end of a line, and one where `</strong>` is on a line on its own. Standardising this across the files being modified such that `</strong>` is in a line on its own would be good.